ui/packages/catalog-ui-search/src/main/webapp/react-component/location/keyword.js

```diff
@@ -73,7 +82,7 @@ class Keyword extends React.Component {
           onChange={option => this.onChange(option)}
           minimumInputLength={2}
           placeholder="Enter a region, country, or city"
-          url="./internal/geofeature/suggestions"
+          suggester={suggester}
```

Comment:
What happens when Autocomplete calls the suggester provided and this call returns a 500?

```java
  async suggester(input) {
    const res = await this.fetch(`./internal/geofeature/suggestions?q=${input}`)
    return await res.json()
  }
```

Reply:
The handling is in a `try` statement. The `catch` displays a relevant `Geo feature endpoint unavailable` message.

Reply:
It seems like only geofeature(id) is wrapped in a try catch